README.md

```diff
@@ -74,9 +74,9 @@ This project is a biotechnology conference web where you can see all the informa
 
 ### Key Features <a name="key-features"></a>
 
-- This is the first itertation of my portfolio.
-- It contains my professional information.
-- All my porjects will be displayed in this werbsite.
+- Full responsive Home and About sections.
+- Featured speakers dynamically created using JavaScript.
+- Mobile Menu functionaluity implemented in JavaScript.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] Kindly publish your website on GitHub Pages, so in this way, a user that wants to know more about your projects can navigate and test it easily! 💯